src/main/java/com/capitalone/dashboard/config/MongoConfig.java

```diff
@@ -57,6 +61,9 @@ public MongoClient mongo() throws Exception {
         MongoClientOptions.Builder builder = new MongoClientOptions.Builder();
         builder.maxConnectionIdleTime(60000);
         builder.sslEnabled(Boolean.parseBoolean(dbssl));
+        // builder.serverSelectionTimeout(30000);   // MongoDB default 30 seconds
```

Comment:
Can we remove commented code please

Reply:
Having the line is a reminder we can change that as well if necessary, as the default is quite long. Anyway, I uncomment it for now